src/components/lib/ConnectWalletDialog.js

```diff
@@ -55,17 +54,15 @@ export const ConnectWalletDialog = ({ shown, onConnected }) => {
 
 	const ChooseWalletStep = () => {
```

Comment:
I think the components `ChooseWalletStep`, `ConnectToWalletStep`, `ConnectionError` should be a functional component instead of creating a component in "ConnectWalletDialog" each time the component is rendered.

Reply:
Addressed in #206 with 81438cb